### DISCUSSION TITLE
Raycast-like Update Flow

### DISCUSSION THREAD

**Discussion by tmm** [ghostty-org/ghostty#2638] [Feature Requests, Ideas] [answered]
Sparkle works great, but could be cool to have a [similar update flow to Raycast](https://medium.com/raycastapp/how-we-built-an-app-update-flow-our-users-love-89820602e8fe), where updates are downloaded in the background and there is some first-class interface for viewing release notes.

**Comment by mitchellh** [comment]
I believe Sparkle lets you customize all of this. It's a bit of a burden because you have to implement a number of different views but I think we can continue to use Sparkle and do this. 

On one hand, I'm very supportive of this, I think a less intrusive update flow is great. On the other hand, the Sparkle flow I've found is very familiar and inspires some confidence in people. Raycast is a bit unique because it doesn't generally have a UI up so a pop-up popping up would be very intrusive. With Ghostty we have a window that is focused so it feels less so.

**Reply by mitchellh** [reply, marked as answer]
#9116
